src/pcapi/routes/serialization/reimbursement_csv_serialize.py

```diff
@@ -67,13 +71,13 @@ def __init__(self, payment_info: namedtuple = None):
             self.user_first_name = payment_info.user_firstName
             self.booking_token = payment_info.booking_token
             self.booking_used_date = payment_info.booking_dateUsed
-            self.booking_total_amount = payment_info.booking_amount * payment_info.booking_quantity
+            self.booking_total_amount = localeNumber(payment_info.booking_amount * payment_info.booking_quantity)
             if payment_info.reimbursement_rate:
-                reimbursement_rate = f"{int(payment_info.reimbursement_rate * 100)}%"
+                reimbursement_rate = localeNumber(f"{int(payment_info.reimbursement_rate * 100)}%")
```

Comment:
Ici on convertit en int, puis en string avec `f""`, puis on passe ça à la fonction qui est sensée convertir un nombre en string. On pourrait peut-être faire ça plus directement avec :

```suggestion
                reimbursement_rate = f"{format_number_as_french(int(payment_info.reimbursement_rate * 100))}%"
```